server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
-	VirtualServerDownloadQuota             uint64  `ms:"virtualserver_download_quota"`
-	VirtualServerUploadQuota               uint64  `ms:"virtualserver_upload_quota"`
-	FileBase                               string  `ms:"virtualserver_filebase"`
-	HostBannerGFXURL                       string  `ms:"virtualserver_hostbanner_gfx_url"`
-	HostBannerURL                          string  `ms:"virtualserver_hostbanner_url"`
-	HostButtonGFXURL                       string  `ms:"virtualserver_hostbutton_gfx_url"`
-	HostButtonToolTip                      string  `ms:"virtualserver_hostbutton_tooltip"`
-	HostButtonURL                          string  `ms:"virtualserver_hostbutton_url"`
-	HostMessage                            string  `ms:"virtualserver_hostmessage"`
-	Name                                   string  `ms:"virtualserver_name"`
-	NamePhonetic                           string  `ms:"virtualserver_name_phonetic"`
-	Password                               string  `ms:"virtualserver_password"`
-	Platform                               string  `ms:"virtualserver_platform"`
-	Status                                 string  `ms:"virtualserver_status"`
-	UniqueIdentifier                       string  `ms:"virtualserver_unique_identifier"`
-	Version                                string  `ms:"virtualserver_version"`
-	WelcomeMessage                         string  `ms:"virtualserver_welcomemessage"`
+	UniqueIdentifier                       string        `ms:"virtualserver_unique_identifier"`
+	Name                                   string        `ms:"virtualserver_name"`
+	WelcomeMessage                         string        `ms:"virtualserver_welcomemessage"`
+	Platform                               string        `ms:"virtualserver_platform"`
+	Version                                string        `ms:"virtualserver_version"`
+	Password                               string        `ms:"virtualserver_password"`
+	HostMessage                            string        `ms:"virtualserver_hostmessage"`
+	FileBase                               string        `ms:"virtualserver_filebase"`
+	HostBannerURL                          string        `ms:"virtualserver_hostbanner_url"`
+	HostBannerGFXURL                       string        `ms:"virtualserver_hostbanner_gfx_url"`
+	HostButtonToolTip                      string        `ms:"virtualserver_hostbutton_tooltip"`
+	HostButtonURL                          string        `ms:"virtualserver_hostbutton_url"`
+	HostButtonGFXURL                       string        `ms:"virtualserver_hostbutton_gfx_url"`
+	MachineID                              string        `ms:"virtualserver_machine_id"`
+	NamePhonetic                           string        `ms:"virtualserver_name_phonetic"`
+	IP                                     string        `ms:"virtualserver_ip"`
+	ServerNickname                         string        `ms:"virtualserver_nickname"`
```

Comment:
you have the context to ensure that its clear as in this will be `server.Nickname` and the other would be `client.Nickname` if you kept the name would stutter e.g. `server.ServerNickname`